internal/cli/opsmanager/logs/decrypt_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build unit
+// +build unit
+
+package logs
+
+import (
+	"encoding/base64"
+	"testing"
+
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/test"
+	"github.com/spf13/afero"
+)
+
+func TestDecryptBuilder(t *testing.T) {
+	test.CmdValidator(
+		t,
+		DecryptBuilder(),
+		0,
+		[]string{
+			flag.File,
+			flag.Out,
+			flag.LocalKeyFile,
+			flag.KMIPClientCertificateFile,
+			flag.KMIPServerCAFile,
```

Comment:
this test is missing `flag.Output`

Reply:
I removed the flag.Output from the decrypt command (since we have flag.Out), so there is no need to have it in this list